shell/platform/darwin/ios/framework/Source/FlutterViewController_Internal.h

```diff
@@ -16,7 +16,7 @@
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterViewResponder.h"
 
 namespace flutter {
-class FlutterPlatformViewsController;
+class PlatformViewsController;
```

Comment:
Thank you. We should reserve the `Flutter` name prefix for where we're forced to, in Obj-C code.